app/uk/gov/hmrc/eeittadminfrontend/connectors/GformConnector.scala

```diff
@@ -79,8 +80,13 @@ class GformConnector(wsHttp: WSHttp, sc: ServicesConfig) {
 
   def deleteTemplate(
     formTemplateId: FormTemplateId
-  )(implicit headerCarrier: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] =
-    wsHttp.DELETE[HttpResponse](gformUrl + s"/formtemplates/$formTemplateId")
```

Comment:
Results.* is a frontend controller concern. Why do we map it here? Ideally, like other methods, the controller code should map the connector response to whatever result is appropriate right?

Reply:
This is [why](https://github.com/hmrc/eeitt-admin-frontend/pull/158/files#diff-d8a788cb12016bfa5392937149edc9faffcc7a97860b455dc63cf0ead33e01fcL244-R237). 

But this is just a quick way on improving things. As a matter of fact delete endpoint on gform side does not work properly and needs to be fixed. So I'd postpone any more changes related to delete code for later.